pages/newanalysis/index.js

```diff
@@ -144,15 +145,22 @@ export default function NewAnalysis({ allEntries, handleAllEntriesChange }) {
           </ChooseTypeOfAnalysisInput>
         </ChooseTypeOfAnalysisFormfield>
         {typeOfAnalysis === "PastAnalysis" ? (
-          <>
-            <PastAnalysisForm typeOfAnalysis={typeOfAnalysis} />
+          <PastAnalysisForm typeOfAnalysis={typeOfAnalysis} />
+        ) : (
+          ""
+        )}
+        {typeOfAnalysis === "InnerSituationAnalysis" ? (
+          <InnerSituationAnalysisForm typeOfAnalysis={typeOfAnalysis} />
+        ) : (
+          ""
+        )}
 
-            <Button variant="big" type="submit">
-              💾 Speichern
-            </Button>
-          </>
+        {typeOfAnalysis !== "" ? (
+          <Button variant="big" type="submit">
+            💾 Speichern
+          </Button>
         ) : (
-          <p>Nothing yet</p>
+          ""
```

Comment:
"nothing" in JSX is best represented as `null`
```suggestion
          null
```

Reply:
thanks - i did not know that 🙏🏻